### ISSUE TITLE
Backstab is too expensive to use effectively

### ISSUE THREAD

**Issue by Wirednuke** [jkachhad/Stormhalter#1482]
**Backstab currently cost too much mana to use effectively.** You are required to maintain hide (3 mana), Stalker in the shadows (26 mana) and Speed (20 mana) in order to backstab which uses 12 mana in itself. The thief has a smaller casting pool than other casters, and it's nearly impossible to use it and still be able to maintain your other spell casting abilities (shadow step, venom, recalls, dark, speed, etc) without a sack of mrp. 

Backstab is an end-game ability and should be quite powerful. It uses 12 mana (32 mana if you venom both daggers) and does significantly less damage than other abilities available to the other casters for a similar cost. An average backstab attack without venom is about 70 damage for 12 mana, compared to a level 16 Ice spear which does over 200, or death that does at least 160. You could even draw comparison to the MA jumpkick ability which does over 100 damage for 1 stamina point, and backstab can't be used 1 hex away.

Proposal:

- Make Backstab (and any future thief special attacks) cost stamina instead of mana. This will allow backstab to be used more frequently and be supplemented with a more readily available resource (stamina pots). 
- If the above isn't possible, it needs to be cheaper to use or have its damage increased significantly.. i.e. maybe 100% crit chance?

Wanted to add as well, that when the spell was broken initially and had no mana cost it was an absolute blast to play with. When I grouped with other players I was able to quickly move and dispatch ranged and archers before they caused harm to the party. I know 0 mana/stamina cost is not an option obviously, but if it's going to have its current damage output it needs to have very low mana/stam requirements in order to be fun/useful. **Currently a backstab won't kill any (appropriate level) mob in 1 shot, so typically follow on attacks are required.** 

---
[View on Discord](https://discord.com/channels/1056432310632841276/1530607259330285660/1530607259330285660)

---
[View on Discord](https://discord.com/channels/577503035526348810/1532139812792959036/1532139812792959036)

**Comment by dedmist** [comment]
My opinion:
I need to get some play hours in with backstab, but I suspect this is correct.  As a hunting skill, I'm already fairly short on mana all the time.  There is a decent volume of mrps that can facilitate this, however, I'm planning on doing a MA run vs a dagger run to see what the utility of jumpkick vs backstab is.  I don't mind the MA run being easier to do, or even have a slightly higher killing rate, but I would like them to be comparable.

I had a few thoughts for encouraging more of a toe-to-toe playstyle.  I think that venom on daggers could be perm for melee, single use for range.  Or maybe perm but with buff timer aura that prevents melee consumption, but when thrown is still consumed.  I think this is a better way to close the gap between dagger damage and heavier hitters by using magic as an aid.  This would also give a boost to backstab by more often having a venom application.  If the numbers are right then backstab is unfavorable to death, a 2 turn skill, and should probably be more comparable to somewhere between 180-200 per 2 turns.  

I think we should reward the more dangerous playstyle.

Shadowstep and Backstab could both benefit from lower magic cost and higher stamina cost.  Maybe 1mp and some stam for each.  It's not the worst thing for a thief to have to play with low magic, but it's a little rough to play with none.

**Comment by Wirednuke** [comment]
![image](https://user-images.githubusercontent.com/98596410/160137365-b95c2466-e4c0-4864-8ce5-9d69528c6583.jpeg)

Added comment for Amanita, with his permission.

**Comment by bsmittyau7** [comment]
I’ve also played numerous hours on a thief, and I rarely ever use backstab simply because of all the built in costs.  There’s very few scenarios I actually needed to backstab something opposed to just using a regular venom rdagger attack, so I guess it’s just too situational for my liking, combined with a cost that doesn’t seem to justify the damage output. -Xero

**Comment by malcies** [comment]
Wanted to start with, LOVE the thief changes in general. Watching some of the early demos from Lux is what brought me over from Lands.

Currently I have a 21/17/9th/15 (dagger) thief in softcore and I absolutely loved where thieves were, not useless, not overpowered, in that middle point - but most of all, sooooooo fun

Backstab felt like that 1 "classic" thief ability that was missing, and when I finally made it to skill 16 I was excited to use it. 
Wasnt sure how it would play out, so I went down to -4 kes, and was ready to make a mino my first victim.... he survived the attack (well, venom got him, but the backstab didnt). on top of that, I was down over 50% of my mana of setting up for that 1 attempt.

Have talked with Brick and other thieves over the months and his proposal is bang on:
1) I would much rather use it more often in its current state, but it would need to become a standard attack, or stamina using (it still requires a LOT of setup vs. say JK)
2) if it stays like it is, it'll need a HUGE damage buff, like, 1 shot anything that isnt a boss or miniboss kinda boost at least

Such a fun class, would love to see some tweaks to thieves ultimate ability :)

**Comment by Wirednuke** [comment]
> I think the problem with this is it's simultaneously too much mana, but also can deal as much as an ice spear while not having as much risk.
> 
> My suggestion would be that daggers, when dual wielded, have a % chance to attack with both weapons depending on skill. Regardless of class. Thieves would still have the best daggers available.
> 
> I think all weapons should have unique attributes or why have so many weapon skills?
> 
> It's also kind of weird to me that bow thieves get hoodwink and half dps of other classes while melee thieves are showered in delicious gifts. Stalker in the shadows is the real winning ability for thieves. I could go on for a while but the general gist is I agree backstab is... missing something.

I agree with your point on bow thieves needing a little love. I don’t enjoy the bow play style too much, so I’ll defer to your interest and expertise here. You should start another issue so we can discuss it.

**Comment by dedmist** [comment]
> It's also kind of weird to me that bow thieves get hoodwink and half dps of other classes while melee thieves are showered in delicious gifts. Stalker in the shadows is the real winning ability for thieves. I could go on for a while but the general gist is I agree backstab is... missing something.

It's the bow risk factor.  I was thinking that we create a shortbow and a longbow that can block and have single round shots.  

The longbow will be unhidden.  It should have blocking somewhere around bbs level.  There needs to be a slightly more dynamic variation on gameplay though, not sure what at this point.  We could probably create this weapon by using the nocked version for the icon and call it a returning weapon; I haven't really looked at the code to see how to make it or if we'll need a core code change.  We probably need a few different longbows, we'll have to determine where we think it's appropriate for a thief to use one and how much fun they should be.  I expect we could have one that procs something interesting like a 1hex PC; my other thought was a retreat + shoot, but I'm not entirely sure how the interface for this should work, I was thinking it would only proc after a successful hit but then it should hit a random target?  Maybe we also have a yeoman's bow that hits like a super heavy weapon - though this should require a nock round.
The target playstyle would be out in the open, better group lair support with enough tactical advantage and movement to help control the situation.

The short bow would be a single round firing as well, but allows for hidden.  It should block less than the longbow and hit harder than a rdagger.  The trick here is to limit the range - so it's basically a melee bow.  I'd like to make it full power at distance 0, 3/4 power at 1, 5/8 power at 2, half power at 3.  The hit percentage should be very high for 2 and 3 distance.
For the short bow, I think it'd be interesting to wield it with an agate.  The agate hits for "distract" which will draw a mob to you, when you're unseen.  You can target the mob directly or on the ground to attract all mobs within sight distance (effectively works like a blink on that hex).  Of course this means that the new long/short bows wouldn't need 2h and the agate is bypassed for blocking checks.  The short bow would probably benefit even more from a retreatfire due to needing to move from stalker.
